src/api/covariates/coral_atlas.py

```
@@ -32,9 +33,13 @@ def _fetch(self, x: float, y: float, radius: float, request_datetime: datetime)
         headers = {
             "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"
         }
-        resp = requests.get(url, headers=headers)
-        status_code = resp.status_code
-        if status_code != 200:
+        for attempt in range(3):
+            resp = requests.get(url, headers=headers)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)